<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>import_cplus</class>
 <widget class="QDialog" name="import_cplus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Importación Contaplús</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Ejercicio:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ejercicioscomboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="anchopushButton">
       <property name="text">
        <string>&amp;Averiguar ancho de cuentas auxiliares</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="title">
        <string>Importar</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="vencicheckBox">
          <property name="text">
           <string>Vencimientos</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="amortcheckBox">
          <property name="text">
           <string>Plan amortizaciones</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="logcheckBox">
       <property name="text">
        <string>Mostrar información en log del proceso</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Ruta:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rutapushButton">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>26</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Cargar fichero documento</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="keme.qrc">
           <normaloff>:/iconos/graf/fileimport.png</normaloff>:/iconos/graf/fileimport.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="rutalineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancelarpushButton">
         <property name="text">
          <string>&amp;Salir</string>
         </property>
         <property name="icon">
          <iconset resource="keme.qrc">
           <normaloff>:/iconos/graf/application-exit.png</normaloff>:/iconos/graf/application-exit.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="procesarpushButton">
         <property name="text">
          <string>&amp;Procesar</string>
         </property>
         <property name="icon">
          <iconset resource="keme.qrc">
           <normaloff>:/iconos/graf/exec.png</normaloff>:/iconos/graf/exec.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="logtextEdit">
     <property name="minimumSize">
      <size>
       <width>480</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ejercicioscomboBox</tabstop>
  <tabstop>anchopushButton</tabstop>
  <tabstop>vencicheckBox</tabstop>
  <tabstop>amortcheckBox</tabstop>
  <tabstop>logcheckBox</tabstop>
  <tabstop>rutapushButton</tabstop>
  <tabstop>rutalineEdit</tabstop>
  <tabstop>cancelarpushButton</tabstop>
  <tabstop>procesarpushButton</tabstop>
  <tabstop>logtextEdit</tabstop>
 </tabstops>
 <resources>
  <include location="keme.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelarpushButton</sender>
   <signal>clicked()</signal>
   <receiver>import_cplus</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
